### PR TITLE
Style Guide fixit day

### LINKED ISSUE #3247: Style Guide (Part 1/4)
Docs to cover:
- files in the `architecture` folder
- files in the `training` folder
- `add-column.md` to `build-a-rust-app-with-cockroachdb.md`

To do:
- Change H2+ to sentence case
- Add `{% include copy-clipboard.html %}` where necessary
- Edit tone for examples (Outlined in [Language and Tone](https://github.com/cockroachdb/docs/blob/master/STYLE.md#language-and-tone): "For tutorials, training, and examples, use the first-person plural tense (e.g., we, let's). These docs should be conversational, as if they are teaching the user." )
- If you are feeling ambitious, use the [prose linter](https://github.com/cockroachdb/docs/pull/3194) in the CLI to find hyperbolic language in your files or from your PR's TC build log, and evaluate whether each instance should be updated accordingly.

### PR DESCRIPTION
**Pull request by lhirata**
Closes #3247.

### CODE REVIEW

File: v2.1/admin-ui-access-and-navigate.md [hunk 1]
```diff
@@ -110,7 +112,8 @@ You can hover over each graph to see actual point-in-time values.
 
 <img src="{{ 'images/v2.1/admin_ui_hovering.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
 
-{{site.data.alerts.callout_info}}By default, CockroachDB stores timeseries metrics for the last 30 days, but you can reduce the interval for timeseries storage. Alternately, if you are exclusively using a third-party tool such as <a href="monitor-cockroachdb-with-prometheus.html">Prometheus</a> for timeseries monitoring, you can disable timeseries storage entirely. For more details, see this <a href="operational-faqs.html#can-i-reduce-or-disable-the-storage-of-timeseries-data">FAQ</a>.
+{{site.data.alerts.callout_info}}
```
Comment (rmloveland) [inline v2.1/admin-ui-access-and-navigate.md]: Why the whitespace gap here?

Comment (rmloveland) [reply v2.1/admin-ui-access-and-navigate.md]: Nevermind, sorry - I think this is an artifact of Github's diff display.

File: v2.1/admin-ui-custom-chart-debug-page.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ Options include:
 
 To compare system vs. userspace CPU usage, select the following values under **Metric Name**:
 
-+ `sys.cpu.sys.percent`
-+ `sys.cpu.user.percent`
+- `sys.cpu.sys.percent`
```
Comment (rmloveland) [inline v2.1/admin-ui-custom-chart-debug-page.md]: FINE THEN LAUREN

File: v2.1/build-a-rust-app-with-cockroachdb.md [hunk 3]
```diff
@@ -54,7 +54,9 @@ Next, use the following code to again connect as the `maxroach` user but this ti
 
 Download the <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/app/txn-sample.rs" download><code>txn-sample.rs</code></a> file, or create the file yourself and copy the code into it.
 
-{{site.data.alerts.callout_info}}With the default <code>SERIALIZABLE</code> isolation level, CockroachDB may require the <a href="transactions.html#transaction-retries">client to retry a transaction</a> in case of read/write contention. CockroachDB provides a generic <strong>retry function</strong> that runs inside a transaction and retries it as needed. You can copy and paste the retry function from here into your code.{{site.data.alerts.end}}
+{{site.data.alerts.callout_info}}
```
Comment (rmloveland) [inline v2.1/build-a-rust-app-with-cockroachdb.md]: Not in scope for this PR, but OMG this should be an incluuuuuuuuuude

File: v2.1/views.md [hunk 4]
```diff
@@ -23,11 +23,12 @@ When you have a complex query that, for example, joins several tables, or perfor
 
 Let's say you're using our [sample `startrek` database](generate-cockroachdb-resources.html#generate-example-data), which contains two tables, `episodes` and `quotes`. There's a foreign key constraint between the `episodes.id` column and the `quotes.episode` column. To count the number of famous quotes per season, you could run the following join:
 
+{% include copy-clipboard.html %}
 ~~~ sql
-> SELECT startrek.episodes.season, count(*)
-  FROM startrek.quotes
-  JOIN startrek.episodes
-  ON startrek.quotes.episode = startrek.episodes.id
+> SELECT startrek.episodes.season, count(*) \
```
Comment (rmloveland) [inline v2.1/views.md]: I think we are *not* using the backslashes, right?

File: v2.1/views.md [hunk 5]
```diff
@@ -44,12 +45,13 @@ Let's say you're using our [sample `startrek` database](generate-cockroachdb-res
...
 Alternatively, to make it much easier to run this complex query, you could create a view:
 
+{% include copy-clipboard.html %}
 ~~~ sql
-> CREATE VIEW startrek.quotes_per_season (season, quotes)
-  AS SELECT startrek.episodes.season, count(*)
-  FROM startrek.quotes
-  JOIN startrek.episodes
-  ON startrek.quotes.episode = startrek.episodes.id
+> CREATE VIEW startrek.quotes_per_season (season, quotes) \
```
Comment (rmloveland) [inline v2.1/views.md]: Same as above re: backslashes

File: v2.1/views.md [hunk 6]
```diff
@@ -101,9 +105,10 @@ Let's say you have a `bank` database containing an `accounts` table:
 
 You want a particular user, `bob`, to be able to see the types of accounts each user has without seeing the balance in each account, so you create a view to expose just the `type` and `email` columns:
 
+{% include copy-clipboard.html %}
 ~~~ sql
-> CREATE VIEW bank.user_accounts
-  AS SELECT type, email
+> CREATE VIEW bank.user_accounts \
```
Comment (rmloveland) [inline v2.1/views.md]: Slashes of back

File: v2.1/views.md [hunk 7]
```diff
@@ -160,28 +169,32 @@ pq: user bob does not have SELECT privilege on table accounts
...
-### Creating Views
+### Creating views
 
 To create a view, use the [`CREATE VIEW`](create-view.html) statement:
 
+{% include copy-clipboard.html %}
 ~~~ sql
-> CREATE VIEW bank.user_accounts
-  AS SELECT type, email
+> CREATE VIEW bank.user_accounts \
```
Comment (rmloveland) [inline v2.1/views.md]: backslashes?

**Comment by lhirata** [comment]
Review status: :shipit: complete! 0 of 0 LGTMs obtained

---

*[v2.1/admin-ui-access-and-navigate.md, line 115 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LELyCQ60ZXZbcEgWvBZ-r7-115:-LEkuyfhBw270y3lHdt-:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/admin-ui-access-and-navigate.md#L115)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Nevermind, sorry - I think this is an artifact of Github's diff display.
</blockquote></details>

Done.

---

*[v2.1/admin-ui-custom-chart-debug-page.md, line 43 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LELyCQ60ZXZbcEgWvB_-r7-43:-LEkuzwH1NllTewreZmX:bb6xktx) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/admin-ui-custom-chart-debug-page.md#L43)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

FINE THEN LAUREN
</blockquote></details>

JEEZ

---

*[v2.1/build-a-rust-app-with-cockroachdb.md, line 57 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LEQAlgPLcFAUtSOdByN-r7-57:-LEkv3VA-v-hfmR6tfIF:b8e6fci) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/build-a-rust-app-with-cockroachdb.md#L57)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Not in scope for this PR, but OMG this should be an incluuuuuuuuuude
</blockquote></details>

Done. Noted for your next docs project

---

*[v2.1/views.md, line 28 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LEQJB7FQcHhBiWLReZ0-r7-28:-LEkvCje-AIhehCVnDiy:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/views.md#L28)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

I think we are *not* using the backslashes, right?
</blockquote></details>

Done.

---

*[v2.1/views.md, line 50 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LEQJB7FQcHhBiWLReZ0-r7-50:-LEkvGVY1Q2zfVAmk1k0:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/views.md#L50)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Same as above re: backslashes
</blockquote></details>

Done.

---

*[v2.1/views.md, line 110 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LEQJB7FQcHhBiWLReZ0-r7-110:-LEkvInB7aCJyEKzQoW1:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/views.md#L110)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Slashes of back
</blockquote></details>

Done.

---

*[v2.1/views.md, line 180 at r7](https://reviewable.io/reviews/cockroachdb/docs/3261#-LEQJB7FQcHhBiWLReZ0-r7-180:-LEkvJGADG7HERMue9fi:b-dz155i) ([raw file](https://github.com/cockroachdb/docs/blob/11b8c0d330792b164a8ce483ac14ced895edd2f4/v2.1/views.md#L180)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

backslashes?
</blockquote></details>

Done. Thanks for the catch!

---


*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/3261)*

**Comment by lhirata** [comment]
TFTR @rmloveland !

**Pull request merged**